src/drive/web/modules/views/OnlyOffice/helpers.js

```diff
@@ -58,6 +58,30 @@ export const makeOnlyOfficeIconByClass = fileClass => {
   return iconByClass[fileClass]
 }
 
+export const makeExtByClass = fileClass => {
+  const extByClass = {
+    text: 'docx',
+    spreadsheet: 'xlsx',
+    slide: 'pptx'
+  }
+
+  return extByClass[fileClass]
+}
+
+export const makeMimeByClass = fileClass => {
```

Comment:
Are they equal to what we have on the stack's side?

Reply:
Je vois avec @nono . En tout cas ce sont ceux officiels MDN

Reply:
Yes, the stack has the same content-types: https://github.com/cozy/cozy-stack/blob/master/model/vfs/vfs.go#L667-L675